planning/obstacle_cruise_planner/src/node.cpp

```diff
@@ -1114,7 +1121,11 @@ ObstacleCruisePlannerNode::createCollisionPointsForOutsideCruiseObstacle(
   const auto collision_points = polygon_utils::getCollisionPoints(
     traj_points, traj_polys, obstacle.stamp, resampled_predicted_path, obstacle.shape, now(),
     is_driving_forward_, collision_index,
-    vehicle_info_.vehicle_width_m + p.max_lat_margin_for_cruise,
```

Comment:
This equation causes the skip of collision check, even if the polygons will collide.
Is it designed behavior?
If so, I'll change the code to fit this behavior